dev-utils/test.js

```diff
@@ -0,0 +1,236 @@
+  var configs = fileList
+    .map(f => {
+      return require(f)
+    })
+    .reduce((acc, cfg) => {
+      if (cfg.length) {
+        return acc.concat(cfg)
+      } else {
+        acc.push(cfg)
+        return acc
+      }
+    }, [])
+
+  console.log('Config Files: \n', fileList.join('\n'))
+  webpack(configs, (err, stats) => {
+    if (err) {
+      console.log(err)
+      cb(err)
+    }
+    if (stats.hasErrors()) console.log('There were errors while building')
+
+    var jsonStats = stats.toJson()
+    console.log(stats.toString())
+    if (jsonStats.errors.length > 0) {
+      jsonStats.errors.forEach(function (error) {
+        console.log('Error:', error)
+      })
+      cb(jsonStats.errors)
+    } else {
+      cb()
+    }
+  })
+}
+
+function onExit (callback) {
+  function exitHandler (err) {
+    try {
+      callback(err)
+    } finally {
+      if (err) console.log(err)
+    }
+  }
+
+  process.on('exit', exitHandler)
+
+  process.on('SIGINT', exitHandler)
+
+  process.on('uncaughtException', exitHandler)
+}
+
+function startSelenium (callback, manualStop) {
+  callback = callback || function () {}
+  var selenium = require('selenium-standalone')
+  var drivers = {
+    chrome: {
+      version: '2.38',
+      arch: process.arch,
+      baseURL: 'https://chromedriver.storage.googleapis.com'
+    },
+    firefox: {
+      version: '0.19.1',
+      arch: process.arch
+    }
+  }
+  selenium.install(
+    {
+      logger: console.log,
+      drivers: drivers
+    },
+    function (installError) {
+      if (installError) {
+        console.log('Error while installing selenium:', installError)
+      }
+      selenium.start({ drivers: drivers }, function (startError, child) {
+        function killSelenium () {
+          child.kill()
+          console.log('Just killed selenium!')
+        }
+        if (startError) {
+          console.log('Error while starting selenium:', startError)
+          return process.exit(1)
+        } else {
+          console.log('Selenium started!')
+          if (manualStop) {
+            callback(killSelenium)
+          } else {
+            onExit(killSelenium)
+            callback()
+          }
+        }
+      })
+    }
+  )
+}
+
+function runE2eTests (configFilePath, runSelenium) {
+  // npm i -D selenium-standalone webdriverio wdio-jasmine-framework
+  var Launcher = require('webdriverio').Launcher
+  var wdio = new Launcher(configFilePath)
+  function runWdio () {
+    wdio.run().then(
+      function (code) {
+        process.stdin.pause()
+        process.nextTick(() => process.exit(code))
+        // process.exit(code)
+      },
+      function (error) {
+        console.error('Launcher failed to start the test', error)
+        process.stdin.pause()
+        process.nextTick(() => process.exit())
+        // process.exit(1)
+      }
+    )
+  }
+  if (runSelenium) {
+    startSelenium(runWdio)
+  } else {
+    runWdio()
+  }
+}
+
+module.exports = {
+  runUnitTests: runUnitTests,
```

Comment:
nit: Use object shorthand properties. Can you also add https://eslint.org/docs/rules/object-shorthand to eslint rules

Reply:
Let's address this in a separate PR.

Reply:
Sure